src/core/utils/i18n.js

```diff
@@ -0,0 +1,35 @@
+/* @flow */
+import type { I18nType } from 'core/types/i18n';
+
+type GetLocalizedTextWithLinkPartsParams = {|
+  i18n: I18nType,
+  text: string,
+  linkStart?: string,
+  linkEnd?: string,
+|};
+
+export const getLocalizedTextWithLinkParts = ({
+  i18n,
+  text,
+  linkStart = 'linkStart',
+  linkEnd = 'linkEnd',
+}: GetLocalizedTextWithLinkPartsParams): Object => {
+  const linkDelimiter = '__LINK__';
+
+  const localizedExploreMoreLink = i18n.sprintf(text, {
+    [linkStart]: linkDelimiter,
+    [linkEnd]: linkDelimiter,
+  });
+
+  const parts = localizedExploreMoreLink.split(linkDelimiter);
+
+  if (parts.length === 1) {
+    throw new Error('linkStart and linkEnd values cannot be missing from text');
```

Comment:
Could you add a test for this as well?

Reply:
I do have a question about this? Could I have used invariant for this as well?  I assuming a test is required either way?

Reply:
I'll stick with throw for now but I was just wondering..

Reply:
Good question. I suppose we could use `invariant` here. I'm not really sure how we decide whether to use that or not. I know the goal is to eventually have all of the messages stripped out from `invariant` calls in production, but we haven't actually implemented that yet.

I think `invariant` is appropriate here, as this is an error that should only ever be the result of a mistake in development, and not as the result of something a user did, so therefore we don't need to display a meaningful message to the user.

Reply:
okay, thanks for the info. I guess I'll use invariant here instead since as you said it should only come up during development.